site/source/user-manual/backups/backup-setup/backup-physical.rst

```diff
@@ -0,0 +1,38 @@
+.. _backup-physical:
+
+=================================
+Backup to Secondary Embassy Drive
+=================================
+
+The recommended method of backup is to setup a :ref:`LAN Shared Folder<backup-setup>`.  However, you may use a powered external drive, or external drive via a powered USB 3.0 hub, connected directly to your Embassy if you prefer.
+
+.. warning:: This drive MUST have external power, or connected via a powered USB hub in order to prevent any data corruption due to power constraints!
+
+.. tip:: You can backup to a physical drive using the recommended method above, rather than plugging the drive directly into the Embassy, which can cause problems.
```

Comment:
This tip doesn't really make sense here given the introductory paragraph which basically says the same thing.